charts/hivemq-operator/values.yaml

```diff
@@ -77,21 +77,23 @@ monitoring:
 hivemq:
   # Custom container spec to add to the HiveMQ Pod
   sidecars: []
-  # Custom init container specs to add to the HiveMQ Pod. This is an extension of the initialization field. In comparison this field does not have any defaults but allows for more granular configuration using the full K8s Container API
+  # Custom init container specs to add to the HiveMQ Pod. This is an extension of the initialization field. In comparison, this field does not have any defaults but allows for more granular configuration using the full K8s Container API
   initContainers: []
   # HiveMQ Pod security context
+  # To configure see example https://github.com/hivemq/helm-charts/tree/master/container/README.md
   podSecurityContext: {}
-    # fsGroup: 1001
+    # fsGroup: 10000
     # runAsNonRoot: true
-    # runAsUser: 1000
+    # runAsUser: 10001
     # sysctls:
       # To ensure we don't run into port exhaustion
       # - name: net.ipv4.ip_local_port_range
       #   value: "1024 65535"
   # HiveMQ container security context within the Pod
   containerSecurityContext:
     # runAsNonRoot: true
-    # runAsUser: 1000
+    # runAsGroup: 10000
```

Comment:
Lets keep runAsGroup out of the values.yaml 
In my opinion that becomes a special case that we deal with in the documentation and from there point to your example dockerfile